src/handwritten/_lib_time.py

```diff
@@ -29,21 +30,30 @@ class AbsoluteTime(ctypes.Structure):
     MAX_FS = 10**9
     MAX_YS = 10**9
 
+    _EPOCH_1904 = ht_datetime(1904, 1, 1, tzinfo=timezone.utc)
+
+    def _convert_to_desired_timezone(self, expected_time_utc: Union[std_datetime, ht_datetime], tzinfo: Optional[timezone] = None):
```

Comment:
I think this method is identical in grpc and lib, so you could move it to a new file, `_time.py`.

Reply:
Except maybe it isn't required at all, see next comment.

Reply:
Moved to _time.py